docs/user_guide/explanations/reconstructions/center_of_rotation.rst

```diff
@@ -12,32 +12,63 @@ However, sometimes the values they find are not accurate, so there is also a man
 to adjust the center of rotation.
 
 The algorithms used for reconstruction require a COR for each sinogram,
-i.e. a list of CORs equal to the number of sinograms (also equal to the number of rows in a projection).
-This is abstracted away and the two final values that are used are: center of rotation and tilt.
+i.e., a list of CORs equal to the number of sinograms (also equal to the number of rows in a projection).
+This is abstracted away, and the two final values that are used are: center of rotation and tilt.
 
-In that case the COR refers to the value at sinogram 0 (also referred to as slice 0).
-Using the tilt we then use a linear regression to calculate the COR for each sinogram in the data.
+In that case, the COR refers to the value at sinogram 0 (also referred to as slice 0).
+Using the tilt, we then use a linear regression to calculate the COR for each sinogram in the data.
 
+What is the Center of Rotation (COR)?
+-------------------------------------
 
-Entering known COR and tilt
+The **Center of Rotation (COR)** is a reference point that defines the axis around which the object rotates
+during image acquisition in tomography. It is the midpoint of the rotational axis when looking at
+a slice of the data (a sinogram).
+
+Why COR Matters:
+  - If the COR is incorrect, the projections will not align correctly during reconstruction,
+    leading to artifacts and blurred images.
+  - COR is specific for each sinogram (i.e., each slice of the object being scanned).
+
+How COR is Used:
+  - For accurate reconstruction, the COR for the first sinogram (sinogram 0, also called slice 0) is determined.
+  - Using the **Tilt** parameter, CORs for subsequent sinograms are calculated through a linear adjustment.
+
+What is Tilt?
+-------------
+
+**Tilt** describes the gradual change in the COR along the axis perpendicular to the rotation plane
+(typically corresponding to the rows in a projection). It reflects any misalignment or non-uniformity
+in the system during the acquisition of the sinograms.
+
+Why Tilt Matters:
+  - Tilt accounts for the variation in COR values across sinograms due to physical or mechanical imperfections.
+  - Without considering tilt, the reconstructed slices may not align correctly, causing distortions or streaking artifacts.
+
+How Tilt is Used:
+  - Tilt is measured as the rate of change of COR across the sinograms.
+  - With the known COR of the first sinogram and the tilt value, the COR for every sinogram
+    is interpolated or extrapolated.
```

Comment:
I think the why and how are a bit redundant with the above.

But it would be useful here to say

- tilt is measure in degrees
- tilt of zero means that the rotation axis is vertical, and the center of rotation is the same for every sinogram

Reply:
It would also be useful to say that the GUI also shows "Slope", which is how far the COR move from one slice to the next.

Reply:
To be more clear. Its good to have the how and why for COR, but I think repeating it for tilt is not so useful.